src/components/CCIP/Search/Search.tsx

```diff
@@ -171,7 +176,12 @@ function Search({ chains, tokens, small, environment, lanes }: SearchProps) {
                         {lane.sourceNetwork.name} {">"} {lane.destinationNetwork.name}
                         {!small && (
                           <span>
-                            {lane?.lane?.supportedTokens ? Object.keys(lane.lane.supportedTokens).length : 0} tokens
+                            {lane?.lane?.supportedTokens ? Object.keys(lane.lane.supportedTokens).length : 0}{" "}
+                            {lane?.lane?.supportedTokens
```

Comment:
This is incorrect.

before it would print "0 token", while now it just show "token if "supportedTokens i false, and it never checks if length is 0